lib/sqlitex/sql_builder.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Sqlitex.SqlBuilder do
```

Comment:
I love this SqlBuilder module.  Very nice work.  I regret that I won't get a chance to test it until later this week.  We should definitely think about expanding this in the future for other SQL commands, but that doesn't need to happen before this merge.